iwyu_globals.cc

```diff
@@ -224,7 +232,31 @@ int CommandlineFlags::ParseArgv(int argc, char** argv) {
       case 'm': mapping_files.push_back(optarg); break;
       case 'n': no_default_mappings = true; break;
       case 'o': no_comments = true; break;
-      case 'u': update_comments = true; break;
+      case 'u': 
+        // As update_comments overrides comment verboseness, there is no break here
```

Comment:
Let's rewrite this section as simply:
```
        update_comments = true;

        // Fall through to comment style parsing unless no value was provided
        if (!optarg)
          break;

        LLVM_FALLTHROUGH;
```

I don't think we need to explain fallthrough, just why we're sometimes avoiding it.

But looking at this it makes me wonder -- do we really need separate processing? `comments_with_namespace` is initialized to `false` in the constructor so if we removed the overwriting below if optarg is NULL, wouldn't it just work? Maybe I'm missing some other weird interaction :-)